### PR TITLE
Update Plumo docs and landing page

### PR DESCRIPTION
**Pull request by kobigurk**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/docs/celo-codebase/protocol/plumo.md [hunk 1]
```diff
@@ -71,25 +71,28 @@ The instructions below are for the first phase of the Plumo setup, Powers of Tau
...
 
-The first step to participating is generating your Celo address.
+The first step to participating is generating your Celo address in a secure location.
 
 * Download the `generate` binary corresponding to your OS from [here](https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0-benchmarking.4).
 * Run it in a command line - navigate in the command line to the relevant folder - and follow the instructions.
     * When asked to `Enter some entropy for your Plumo seed:`, you can use any source of entropy.
     * Make sure to save your passphrase - you will need it later.
-* Send the address generated to kobi@clabs.co. Keep the resulting `plumo.keys` file.
+* Send the address generated to kobi@clabs.co.
```
Comment (aaronmgdr) [inline packages/docs/celo-codebase/protocol/plumo.md]: should we create a special plumo@clabs.co that forwards?

File: packages/docs/celo-codebase/protocol/plumo.md [hunk 2]
```diff
@@ -71,25 +71,28 @@ The instructions below are for the first phase of the Plumo setup, Powers of Tau
...
 Next you'll download the `contribute` binary and begin contributing to the Plumo setup. 
 
 cLabs is running the coordinator server, which has a list of approved participant addresses. Additionally, cLabs is running a few verifiers that verify contributions on-demand, allowing the setup to make progress.
 
-* After receiving confirmation from cLabs, download the `contribute` binary from [here]https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0-rc.1).
-* Run it in the same directory as the `plumo.keys` file as follows:
-    * Windows users: `contribute-windows --coordinator-url https://plumo-setup-phase-1.azurefd.net`
-    * macOS users: `./contribute-macos --coordinator-url https://plumo-setup-phase-1.azurefd.net`
-    * Linux users: `./contribute-linux --coordinator-url https://plumo-setup-phase-1.azurefd.net`
+* After receiving confirmation from cLabs, download the `contribute` binary from [here]https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0).
```
Comment (aaronmgdr) [inline packages/docs/celo-codebase/protocol/plumo.md]: missing a `(` after `[here]`

Comment (pranaymohan) [inline packages/docs/celo-codebase/protocol/plumo.md]:
```suggestion
* After receiving confirmation from cLabs, download the `contribute` binary from [here](https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0).
```

File: packages/docs/celo-codebase/protocol/plumo.md [hunk 3]
```diff
@@ -71,25 +71,28 @@ The instructions below are for the first phase of the Plumo setup, Powers of Tau
...
+* Run it as follows, assuming your keys reside in KEYS_PATH:
+    * Windows users: `./contribute-windows.exe --keys-path KEYS_PATH`
+    * macOS users: `./contribute-macos --keys-path KEYS_PATH`
+    * Linux users: `./contribute-linux --keys-path KEYS_PATH`
```
Comment (pranaymohan) [inline packages/docs/celo-codebase/protocol/plumo.md:93]:
```suggestion
* Run it as follows, replacing the env var $KEYS_PATH with the location of your keys:
    * Windows users: `./contribute-windows.exe --keys-path $KEYS_PATH`
    * macOS users: `./contribute-macos --keys-path $KEYS_PATH`
    * Linux users: `./contribute-linux --keys-path $KEYS_PATH`
```

File: packages/web/public/static/locales/en/plumo.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "pageTitle": "Espero: the Plumo Ceremony",
+  "pageTitle": "The Plumo Ceremony",
```
Comment (pranaymohan) [inline packages/web/public/static/locales/en/plumo.json:2]: are we calling it the Plumo ceremony moving forward?

**Review by pranaymohan — COMMENTED** [review]
just a few nits, looks good!

File: packages/docs/celo-codebase/protocol/plumo.md (hunk 1 above)
Comment (kobigurk) [reply packages/docs/celo-codebase/protocol/plumo.md]: I think that would be awesome. Is it you who can create or should we ask someone else?

File: packages/web/public/static/locales/en/plumo.json (hunk 4 above)
Comment (kobigurk) [reply packages/web/public/static/locales/en/plumo.json:2]: I believe so. Espero, in my mind, is the setup system, which can be used for Plumo and other SNARKs as well.

File: packages/docs/celo-codebase/protocol/plumo.md (hunk 3 above)
Comment (kobigurk) [reply packages/docs/celo-codebase/protocol/plumo.md:93]: Not sure about this. Are env vars convenient for non-developers?

Comment (pranaymohan) [reply packages/docs/celo-codebase/protocol/plumo.md:93]: We could include a line that helps them set it eg.
`export $KEYS_PATH = "/mnt/plumo/plumo.keys"`

But no worries, can skip if more confusing

**Review by pranaymohan — APPROVED** [review]

File: packages/docs/celo-codebase/protocol/plumo.md (hunk 1 above)
Comment (aaronmgdr) [reply packages/docs/celo-codebase/protocol/plumo.md]: hmm its not me. the security team might know who does it

Comment (pranaymohan) [inline packages/docs/celo-codebase/protocol/plumo.md]:
```suggestion
* Send the address generated to plumo@celo.org.
```

**Pull request merged**